crates/node_binding/binding.d.ts

```diff
@@ -12,6 +12,7 @@ export interface ThreadsafeNodeFS {
   writeFile: (...args: any[]) => any
   mkdir: (...args: any[]) => any
   mkdirp: (...args: any[]) => any
+  rm: (...args: any[]) => any
```

Comment:
You need to update this file by running `pnpm -w run build:cli:debug`

Reply:
done